.circleci/config.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  node:
+    version: 8.0.0
```

Comment:
Take a look where we went to try to solve it 
```yml
version: 2
image: ubuntu:16.04
jobs:
  build:
    machine: true

    working_directory: ~/repo

    steps:
      - checkout

      - restore_cache:
          keys:
          - webapp-dependencies-{{ checksum "webapp/package.json" }}
          - webapp-dependencies-

      - run: |
          sudo apt remove docker docker-engine docker.io
          cd /tmp
          wget https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz
          tar xvzf docker-17.06.2-ce.tgz
          sudo cp docker/* /usr/bin/
          sudo mkdir -p /etc/docker
          echo '{ "hosts": ["unix:///var/run/docker.sock"] }' | sudo tee -a /etc/docker/daemon.json
          sudo dockerd &
          sleep 5
          docker info

      - run: |
          export NVM_DIR=/opt/circleci/.nvm
          source /opt/circleci/.nvm/nvm.sh
          nvm install 8
          nvm use 8
          npm install -g yarn
          cd ~/repo/webapp
          yarn
          yarn lint
          yarn build
          yarn test
          yarn storybook > /dev/null 2>&1 &
          sleep 10
          yarn shooting-stars
      - save_cache:
          paths:
            - webapp/node_modules
          key: webapp-dependencies-{{ checksum "package.json" }}
```

Reply:
Please ignore this, this will use circle ci v1 for running react native stuff on a macos slave.